src/Reflection/DocBlock/Tag/LicenseTag.php

```diff
@@ -7,30 +7,24 @@
 
 class LicenseTag implements TagInterface
 {
-    /** @var string */
+    /** @var string|null */
     protected $url;
 
-    /** @var string */
+    /** @var string|null */
     protected $licenseName;
 
-    /**
-     * @return string
-     */
+    /** @return 'license' */
     public function getName()
     {
         return 'license';
     }
 
-    /**
-     * Initializer
-     *
-     * @param  string $tagDocblockLine
-     */
-    public function initialize($tagDocblockLine)
+    /** {@inheritDoc} */
```

Comment:
Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.

Reply:
dd68158de612598d53a7f3dcc96c27c5b3816dbf